PlanningSceneInterface::setStartState(moveit_msgs::RobotState) crashes if there is no "current" robot state.

### Description

Hello,

I'm using Moveit in a situation without a robot, simulated or otherwise; I'd like to plan paths between hypothetical start-and-end states. I only have the URDF/SRDF model, so the notion of the "current" robot position is undefined.

In the implementation of `PlanningSceneInterface::setStartState(moveit_msgs::RobotState)`, note the line `impl_->getCurrentState(rs);` (https://github.com/ros-planning/moveit/blob/master/moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp#L1611)

This means that, even if the `moveit_msgs::RobotState` fully defines the current robot state, this method call fails with an error like this: `[ERROR] [1626532077.440248038]: Failed to fetch current robot state`.

### Your environment
* ROS Distro: Noetic
* OS Version: e.g. Ubuntu 20.04
* Binary build. installed through APT.

### Steps to reproduce

Simply call the method referred to using a moveit_msgs::RobotState, with every joint setting defined, but without a source of current robot joint settings.

The cause seems pretty obvious from the source code.

### Expected behaviour

The provided state is used as the start state in planning.

### Actual behaviour

The aforementioned error is produced, because the method assumes that there is a source of "current" robot states.

### Backtrace or Console output

`[ERROR] [1626532077.440248038]: Failed to fetch current robot state`

--- a/moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp
+++ b/moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp
@@ -1611,7 +1611,10 @@ void MoveGroupInterface::stop()
 void MoveGroupInterface::setStartState(const moveit_msgs::RobotState& start_state)
 {
   moveit::core::RobotStatePtr rs;
-  impl_->getCurrentState(rs);
+  if (start_state.is_diff)
+    impl_->getCurrentState(rs);
+  else
+    rs = std::make_shared<moveit::core::RobotState>(getRobotModel());
   moveit::core::robotStateMsgToRobotState(start_state, *rs);
   setStartState(*rs);
 }

